.github/workflows/cve-scan.yaml

```diff
@@ -0,0 +1,22 @@
+name: Vulnerability Scanning
```

Comment:
Should we call the action `Docker CVE Scanning` since it's scanning the docker image only?